README.md

```diff
@@ -2,21 +2,53 @@
 
 A package to help with writing and executing e2e [Appium](http://appium.io) tests.
 
-## <a name='usage'></a>Usage
+## <a name='requirements'></a>Requirments
 
-Install it with:
+The `nativescript-dev-appium` plugin requires:
+* latest version of [Appium](https://github.com/appium/appium/releases)
```

Comment:
Which is the latest version? I mean now is 'x' but tomorrow will the new one be compatible with the plugin. Maybe we should commit to current one.

Reply:
We commit that *the new one* will be compatible with the plugin and avoid a specific version on purpose.